src/extension.ts

```diff
@@ -63,6 +63,7 @@ export async function activate(context: ExtensionContext) {
         }
       });
     });
+  commands.executeCommand("setContext", "metals:enabled", true);
```

Comment:
Could it be disabled when the server is restarting? Since we cannot run those commands then anyway

Reply:
The trick doesn't seem to work with deactivating the commands unfortunately

Reply:
I don't think it's a big deal. If the user triggers a server restart they're aware it's happening and it shouldn't be a big surprise that commands don't work for a little while.